lib/matplotlib/colors.py

```diff
@@ -1427,25 +1427,41 @@ def __init__(self, boundaries, ncolors, clip=False):
             they are below ``boundaries[0]`` or mapped to *ncolors* if they are
             above ``boundaries[-1]``. These are then converted to valid indices
             by `Colormap.__call__`.
+        extend : {'neither', 'both', 'min', 'max'}, optional
```

Comment:
```suggestion
        extend : {'neither', 'both', 'min', 'max'}, default: 'neither'
```
See our doc conventions: https://matplotlib.org/devdocs/devel/documenting_mpl.html#default-values